src/net_processing.cpp

```diff
@@ -578,6 +578,17 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
         xver.set_u64c(XVer::BU_GRAPHENE_MIN_VERSION_SUPPORTED, grapheneMinVersionSupported.Value());
         xver.set_u64c(XVer::BU_GRAPHENE_FAST_FILTER_PREF, grapheneFastFilterCompatibility.Value());
         xver.set_u64c(XVer::BU_XTHIN_VERSION, 2); // xthin version
+
+        size_t nLimitAncestors = GetArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT);
+        size_t nLimitAncestorSize = GetArg("-limitancestorsize", DEFAULT_ANCESTOR_SIZE_LIMIT) * 1000;
```

Comment:
I don't understand why we use the DEFAULT_ANCESTOR_LIMIT here, but further up we're setting the node default to BU_DEFAULT_ANCESTOR_LIMIT...why the need for the two constants?

Reply:
If we decide to bump the default value of these parameters for all BU nodes (like you had proposed in your PR) we'll change DEFAULT... but not BCH_DEFAULT.  This PR needs to know the BCH_DEFAULT... parameters so that it can make the correct assumption about nodes that don't advertise their parameter values in XVERSION.

Reply:
ah right, I got confused and thought the DEFAULT_* was the original and the BCH was actually the BU default...I wonder if you could make that clear somewhere or use a prefix of BU_* for those values?

Reply:
done